consensus/polybft/consensus_runtime.go

```diff
@@ -0,0 +1,851 @@
+package polybft
+
+import (
+	"encoding/hex"
+	"errors"
+	"fmt"
+	"sync"
+	"sync/atomic"
+
+	"github.com/0xPolygon/pbft-consensus"
+	"github.com/0xPolygon/polygon-edge/consensus/polybft/bitmap"
+	bls "github.com/0xPolygon/polygon-edge/consensus/polybft/signer"
+	"github.com/0xPolygon/polygon-edge/consensus/polybft/wallet"
+	"github.com/0xPolygon/polygon-edge/types"
+	hcf "github.com/hashicorp/go-hclog"
+	"github.com/umbracle/ethgo"
+	"github.com/umbracle/ethgo/abi"
+)
+
+const (
+	eventsBufferSize   = 10
+	stateFileName      = "consensusState.db"
+	uptimeLookbackSize = 2 // number of blocks to calculate uptime from the previous epoch
+)
+
+var (
+	// state sync metrics
+	// TODO: Nemanja - accommodate bridge metrics to Edge
+	// totalStateSyncsMeter = metrics.NewRegisteredMeter("consensus/bridge/stateSyncsTotal", nil)
+
+	// errNotAValidator represents "node is not a validator" error message
+	errNotAValidator = errors.New("node is not a validator")
+	// errQuorumNotReached represents "quorum not reached for commitment message" error message
```

Comment:
I think these comments are not needed, they just reiterate the line below, which is clear